doc/classes/RichTextLabel.xml

```diff
@@ -244,7 +251,7 @@
 			<argument index="0" name="line" type="int">
 			</argument>
 			<description>
-				Removes a line of content from the label. Returns [code]true[/code] if the line exists.
+				Removes the [code]line[/code]th line of content from the label. Returns [code]true[/code] if the line exists.
```

Comment:
Forgot to remove some past words: "th line"

Reply:
It's like "the `n`th child". Replace the `line` parameter with the actual integer and it's a normal sentence: Removes the 6th line...

Reply:
Yeah I also wondered if it would be clear enough to most readers. Maybe it should be rephrased to avoid the awkwardness of a non-number cardinal, but not sure what would be better. IINM this is explicitly about the currently display "line-th" line, and not an actual line "index".